Sources/KlaviyoCore/KlaviyoEnvironment.swift

```diff
@@ -111,6 +115,9 @@ public struct KlaviyoEnvironment {
     public var klaviyoAPI: KlaviyoAPI
     public var timer: (Double) -> AnyPublisher<Date, Never>
 
+    public var SDKName: String
```

Comment:
for consistency I might make these methods with default implementations.

Reply:
Changed these but also changed `apiUrl`.